.github/workflows/integration_tests.yml

```diff
@@ -102,6 +82,7 @@ jobs:
         run: echo "GIT_BRANCH=$(echo ${{ github.ref }} | sed 's|/|-|g')" >> $GITHUB_ENV
 
       - name: Upload Gas Report
+        if: ${{ github.ref == 'refs/heads/main' }}
         uses: actions/upload-artifact@v3
         with:
           name: dao-dao-gas-report-${{ env.GIT_BRANCH }}
```

Comment:
This is now the only place we are using github action storage, and it's very small file (now only pushed in main commits), so we will stay in the free tier.